data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,1611 @@
+			`	He quickly continues. "Of course, we can see it out there. But it is like looking through a very thin film." He looks thoughtful. "The current leading theory is that we are in some kind of dimensional bubble. A pocket dimension of sorts. But we have not yet reached a consensus."`
+			choice
+				`	"What happened to us? Did we all see the same thing?"`
+				`	"Where did those feelings and images come from?"`
+			`	Gavriil frowns in thought. "Based on our initial survey, each person had a slightly different experience. We think that everyone had basically the same event, but as it was a series of impressions, images, and feelings, each person picked up on different aspects of it." He gestures take on a confident tone. "Which makes sense, since our life experiences shape our perceptions. Still, as to what it was..." He pauses, gathering his thoughts. "Our best guess is that this was first contact with some sort of entity." He grins at that. "I know, stating the obvious. But anyway, Dusk thinks that it might be some sort of solitary entity, as it did not use words or meanings, despite having access to our minds. As such, it uses internals to convey meaning instead of words." He glances at a map of the system. "There is, however, no indication as to where it came from.`
+			choice
+				`	"How do we get back?"`
+					goto cutpower
+				`	"What happened to the long range communication?"`
+			`	"Well, that is a good question. We have and are running diagnostics on them, and so far they are passing all their checks. The system safeties, isolation mechanisms, security monitoring, tamper proofing, self-destruct systems, backup hardening..." He shakes his head, stymied. "Everything looks in perfect working order. The only problem is that the quantum-pairs at the heart just do not seem to have a pair anymore. In terms of quantum mechanics, it seems that the rest of the universe quite simply does not exist."`
+			choice
+				`	"That seems problematic. How do we get back?"`
+					goto cutpower
+				`	"Did we destroy the universe by powering up this station?"`
+			`	Gavrill smiles with amusement. "It is certainly a possibility. However, the fact that it was, as best we can tell, powered up previously without destroying the universe as far as we can tell suggests that it transported us rather than destroying everything else." He frowns at the communication panel. "Although everything I know about quantum mechanics refuses to countenance something being able to leave the universe so absolutely like this." He smiles again. "But that is the way of science! Finding new things in the universe that tell us that what we know is not so absolute."`
```

Comment:
```suggestion
			`	Gavrill smiles with amusement. "It is certainly a possibility. However, the fact that it was, as best we can tell, powered up previously without destroying the universe as far as we can tell suggests that it transported us rather than destroy everything else." He frowns at the communication panel. "Although everything I know about quantum mechanics refuses to countenance something being able to leave the universe so absolutely like this." He smiles again. "But that is the way of science! Finding new things in the universe that tells us that what we know is not so absolute."`
```